src/Oro/Bundle/PricingBundle/Model/PriceListTreeHandler.php

```diff
@@ -76,11 +76,15 @@ public function __construct(
      * @param Website|null $website
      * @return CombinedPriceList|null
      */
-    public function getPriceList(Customer $customer = null, Website $website = null)
+    public function getPriceList(Customer $customer = null, $website = null)
```

Comment:
> Type Declarations breaks when a website is null.

Type declaration does not break anything

http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration

>  The declaration can be made to accept NULL values if the default value of the parameter is set to NULL.